include/qml_ros2_plugin/ros2.hpp

```diff
@@ -236,11 +236,22 @@ class Ros2QmlSingletonWrapper : public QObject
    *
    * @param type The type of the messages published using this publisher.
    * @param topic The topic on which the messages are published.
-   * @param queue_size The maximum number of outgoing messages to be queued for delivery to subscribers.
+   * @param history_depth The maximum number of outgoing messages to be queued for delivery to subscribers.
    * @return A Publisher instance.
    */
   Q_INVOKABLE QObject *createPublisher( const QString &topic, const QString &type,
-                                        quint32 queue_size = 1 );
+                                        quint32 history_depth = 0 );
```

Comment:
Just copy-pasting what bab270dfdbb6653a1e6af5b916af6d6fe671c4d3 did for `Subscription`.